### PR TITLE
EL-2900 - Contributing Guide

### PR DESCRIPTION
**Pull request by ashley-hunter**
Adding contributing guide, updating some of our other guides to make sure they are up to date,

Let me know if you have any suggestions about the content of the guide.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+Found a bug or thought of a new feature that would benefit others? Well you can help make UX Aspects even better by contributing to the project.
+Contributing is easy, and all fixes are welcome, no matter how small.
+
+### Before you get started
+
+Before you get started it is good to be aware of some of the practices we adhere to. We have a [Developer Standard](https://github.com/UXAspects/UXAspects/blob/develop/guides/developer-standard.md) document that outlines some of the coding best practices we follow. We have several linting tasks that help enforce these standards and they should be run before creating a Pull Request.
+
+### How to contribute
+
+1. Fork the UX Aspects repository
```
Comment (donna-marijne) [inline CONTRIBUTING.md]: Perhaps we should mention that this is done via GitHub, for those who are not familiar with git.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+2. Clone your forked repository
+3. Make your fix/changes
+4. Run our linting tasks using `grunt lint`.
```
Comment (donna-marijne) [inline CONTRIBUTING.md]: Running the tests should also be part of this, `grunt e2e`.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+5. Commit and Push your changes
+6. Create a Pull Request
```
Comment (donna-marijne) [inline CONTRIBUTING.md]: Perhaps also mention that this is a GitHub step, via the "New pull request" button in the UXAspects repository.

**Comment by ashley-hunter** [comment]
Changes made

**Pull request merged**